CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# v5.0.0 (Thu Jan 02 2025)
+
+#### 💥 Breaking Change
+
+- feat: upgrade to @faker-js/faker v9
+
+This release upgrades to Faker v9, which introduces several breaking changes:
+
+- **Node.js v18+ required**: Faker v9 requires Node.js v18 or higher
+- **TypeScript v5+ required**: Faker v9 requires TypeScript v5 or higher
+- **Different mock values**: Due to Faker v9's improved RNG (53-bit vs 32-bit), generated mock values will be different even with the same seed
+- **Removed deprecated code**: Faker v8 had deprecated many methods, [which have now been removed](https://fakerjs.dev/guide/upgrading.html#removals-of-deprecated-code)
+
+For more details, see the [Faker v9 migration guide](https://fakerjs.dev/guide/upgrading.html).
+
+#### Authors: 1
+
+- Brendan Mulholland ([@bmulholland](https://github.com/bmulholland))
```

Comment:
Rollback this, it will be handled by our versioning tool

However, could you move all this detail inside the PR description. I will reuse the PR description when merging in order to have this level of detail in the changelog